### PR TITLE
Add Docker stats to runtime metrics article

### LINKED ISSUE #13170: Runtime metrics topic doesn't mention `docker stats`
see http://docs.master.dockerproject.com/articles/runmetrics/

### PR DESCRIPTION
**Pull request by ankushagarwal**
Fixes #13170

### CODE REVIEW

File: docs/articles/runmetrics.md [hunk 1]
```diff
@@ -11,6 +11,24 @@ weight = 4
 
 # Runtime metrics
 
+## Docker stats
```
Comment (moxiegirl) [inline docs/articles/runmetrics.md]: Thanks @ankushagarwal for the tweak.  Some adjustments, active voice, case and I think  "running" is implied by "runtime" and "currently" is not needed as "supports" says it all.

--> https://gist.github.com/moxiegirl/59ee4f1d32d9aec54a0b

You can use the `docker stats` commadn to live stream a container's runtime metrics. The command supports CPU, memory usage,
memory limit, and network IO metrics.

Comment (ankushagarwal) [inline docs/articles/runmetrics.md]: Updated. I'll keep these in mind for future PRs

**Comment by moxiegirl** [comment]
Thanks @ankushagarwal LGTM

ping @thaJeztah @docker/docs-owners for the merge.

**Comment by thaJeztah** [comment]
LGTM! thanks Ankush!

**Comment by moxiegirl** [comment]
cherry-picked

**Pull request merged**